### PR TITLE
feat: orderabled views

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |n|
|New feature?   |y|	
|BC breaks?     |n|	
|Deprecations?  |y|	
|Fixed tickets  |n|	

Refactor views and reorder them

# Deprecations

- route nature.reorder
- reorder content type action if the order field is defined
- searches twig filter
- ems_core.application_menu_controller parameter

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -36,7 +36,6 @@ class Configuration implements ConfigurationInterface
     public const USER_LOGOUT_ROUTE = 'fos_user_security_logout';
     public const USER_REGISTRATION_ROUTE = 'fos_user_registration_register';
     public const ADD_USER_ROUTE = 'user.add';
-    public const APPLICATION_MENU_CONTROLLER = null;
```
Comment (Davidmattei) [inline src/DependencyInjection/Configuration.php:39]: 👍

File: src/Form/Field/PdfSizeType.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
```
Comment (Davidmattei) [inline src/Form/Field/PdfSizeType.php:2]:
```suggestion
declare(strict_types=1);
```

**Review by Davidmattei — CHANGES_REQUESTED** [review]
Awesome job! 💯

File: src/Form/Field/OrientationType.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+
```
[the remark this answers is not in the document]
Comment (theus77) [reply src/Form/Field/OrientationType.php:2]: c06235df692d1c37288049afc3053fb6c398a329

File: src/Form/Field/PdfSizeType.php (hunk 2 above)
Comment (theus77) [reply src/Form/Field/PdfSizeType.php:2]: c06235df692d1c37288049afc3053fb6c398a329

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**